server/config/topic_page/equity/CA.textproto

```diff
@@ -136,21 +121,6 @@ categories {
   blocks {
     title: "With disabilities, by sex"
     description: "Percentage of the population of each sex with disabilities"
-    # TODO(beets): This isn"t available at the country level
```

Comment:
Even though we don't have WithHealthInsurance, the next block includes WithNoHealthInsurance which conveys the same thing